.github/workflows/main.yml

```
@@ -22,10 +22,15 @@ jobs:
         # https://hexdocs.pm/elixir/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp
         include:
           # Newest supported Elixir/Erlang pair.
+          - elixir: '1.18.4'
```

there's not yet an Elixir precompiled against OTP 27, however, since https://github.com/erlef/setup-beam/pull/330 this will work:

```suggestion
          - elixir: '1.18.4-otp-27'
```